def print_exception(ex: Exception):
    ...
